library/src/main/java/com/novoda/downloadmanager/RoomFileDao.java

```diff
@@ -15,4 +15,8 @@
 
     @Query("SELECT * FROM RoomFile WHERE RoomFile.batch_id = :batchId")
     List<RoomFile> loadAllFilesFor(String batchId);
+
+    // Only to be used from the UnlinkedDataRemover
```

Comment:
I think this comment is unnecessary, `loadAllFilesFor` is also used in one place. It's fine because we don't expose the database to clients.